app/src/main/res/drawable/ic_broken_image.xml

```diff
@@ -1,17 +1,17 @@
 <!--
-  Copyright (C) 2022 The Android Open Source Project
-
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
+  ~ "Copyright (C) 2023 The Android Open Source Project
```

Comment:
seeing the same issue this consistently makes me worried? I think you were using the settings to automatically add the licences, can you double check your settings?